gui/package-lock.json

```diff
@@ -13188,17 +13187,6 @@
         "node": ">=0.10.0"
       }
     },
-    "node_modules/is-wsl": {
```

Comment:
this I am not sure about.. was never references in the code but clearly points to wsl